tests/pxScene2d/test_imagecache.cpp

```diff
@@ -55,6 +56,25 @@ extern bool continueDownloadHandleCheck;
 // it doesn't guarantee that the returned memory will be available.
 // #define UNAVAILABLE_MEMORY_TEST_ENABLED
 
+bool ignoreNetworkCall = false;
+typedef CURLcode (*curl_perform_t)(CURL*);
+CURLcode curl_easy_perform(CURL * easy_handle )
```

Comment:
Let's try to do another way instead of override a curl method.  Can we use javascript tests instead of native c++ tests to validate functionality?  If the below tests that are commented out are already covered by testRunner tests then let's remove the commented out code.  However, if they are not covered by testRunner tests then can we create testRunner tests for them instead and then remove the commented out tests?